src/extension/src/local_loggers/FileLogger.py

```diff
@@ -74,6 +74,7 @@ def write(self, message, fail_silently=True):
         try:
             if self.log_file_handle is not None:
                 self.log_file_handle.write(message)
+                self.flush()
```

Comment:
Do this wherever we're at risk for data not being written.

Reply:
(For flush, also include (in the flush function), function call to push to disk. (Will share)